source/common/common/byte_order.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#ifdef __APPLE__
+#include <libkern/OSByteOrder.h>
+
```

Comment:
Fix format added it for me. Perhaps a newline-after-last-include thing? I can see if it's somehow specific to my env, but I used the CI image to run fix_format to avoid that very problem.